guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,389 @@
+
+For e.g: Alice can simply post a static address on her website, and receive bitcoin donations at unique on-chain addresses. The static address itself never shows up on-chain.
+
+Silent payments also allow users to customize their static address with labels that are detected when payments are received. This is a powerful tool that provides actionable information for [coin selection](/guide/how-it-works/silent-payments/#coin-selection) with minimal manual effort.
+
+Static addresses & labels enable robust [contacts](/guide/how-it-works/silent-payments/#contacts) features. The result is a better interaction model centred around users.
+
+
+### Rationale
+
+Since the bitcoin blockchain is public, reusing an on-chain address informs the network that these payments are made to the same user. However, specifying a new address for each transaction usually requires user interaction. This is error-prone, takes time and requires manual effort.
+
+Silent payments circumvent both these issues by using static addresses (starting with `sp1`).
+
+## How silent payments work
+
+Silent payments are made in 4 broad steps:
+1. The receiver shares/publishes a static payment address (manual, one-time)
+2. The sender obtains & uses it to derive a unique on-chain address (partly automated)
+3. The sender broadcasts a transaction that pays this derived address (manual)
+4. The receiver scans the blockchain & identifies their payments (automated)
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work-mobile@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "Alice posts a static address on the website of her non-profit, Chasing Rabbits. Bob uses it to send bitcoin from his wallet. This transaction pays a derived on-chain address that Alice can recognise while scanning the blockchain."
+   width = 1024
+   height = 640
+   layout = "full-width"
+%}
+
+## A new conceptual model
+
+Silent payments introduce a new technical model that separates payment information based on their purpose.
+
+For example: the static address is only shared with senders who use it to make payments. Payment detection is done with a separate scan key and doesn't use the static address at all. Here are the main components of this new model:
+
+| Component       |Used by         | Purpose |
+|:-------------|:------------------|:------|
+| Static address | Senders | Making payments
+| Scan key     | Nodes & [light clients](/guide/how-it-works/nodes/#light-nodes)  | Detecting payments
```

Comment:
```suggestion
| Scan key     | [Nodes & light clients]({{ '/guide/how-it-works/nodes/' | relative_url }}) | Detecting payments
```
We're linking to the "Nodes" page, so it makes sense to include that word in the link as well.

Can you please update the internal links on this page to use this `relative_url` format? It's what makes it possible to host the site on any domain/subdomain/etc.